enterprise/reporting/reporting.go

```diff
@@ -584,6 +585,8 @@ func (r *DefaultReporter) Report(metrics []*types.PUReportedMetric, txn *sql.Tx)
 		if err != nil {
 			return fmt.Errorf("executing statement: %v", err)
 		}
+
+		eventStats.Report(workspaceID, &metric)
```

Comment:
This workspaceID is wrong for all multitenant deployments (e.g. pro)... we should probably get rid of it